query/benchmarker_test.go

```diff
@@ -323,3 +336,27 @@ func (mp *mockProcessor) Init(workerNum int) { mp.initCalled = true }
 func (mp *mockProcessor) ProcessQuery(q Query, isWarm bool) ([]*Stat, error) {
 	return mp.processRes, mp.processErr
 }
+
+func Test_getRateLimiter(t *testing.T) {
```

Comment:
one last nit: `TestGetRateLimiter` to match format we already have

Reply:
solved in this new commit. thanks for the fast iterations over the changes :)